client/js/store/modules/auth.js

```diff
@@ -1,8 +1,36 @@
-import { io } from '../../plugins/sails.js';
+import { io, reconnectSockets } from '../../plugins/sails.js';
 import { ROUTE_NAME_LOBBY, ROUTE_NAME_GAME } from '@/router';
 
 import { getPlayerPnumByUsername } from '_/utils/game-utils.js';
 
+async function handleLogin(context, username, password, signup = false) {
+  const type = signup ? 'signup' : 'login';
+  try {
+    const response = await fetch(`/user/${type}`, {
```

Comment:
nit: I think this would be more clear as `authType`
```suggestion
  const authType = signup ? 'signup' : 'login';
  try {
    const response = await fetch(`/user/${authType}`, {
```